test/sauce-sample.json

```diff
@@ -0,0 +1,4 @@
+{
+    "username": "yourSauceUsername",
```

Comment:
The `username` and `accessKey` can be changed to specific user details so that the tests can be run through SauceLabs via their account.